package.json

```diff
@@ -419,6 +418,7 @@
     "jest": "26.6.3",
     "jest-environment-jsdom": "^26",
     "jsonwebtoken": "9.0.0",
+    "prettier": "3.0.2",
```

Comment:
Why did you change this? This may be the cause to one failed workflow. I would suggest that you revert this change, if you do not know what it does.